documentation/Installation.md

```diff
@@ -1477,11 +1488,14 @@ By default, the installer uses the following parameters:
 |podPidsLimit|4096|
 |maxPods|110|
 |cgroupDriver|systemd|
+|serializeImagePulls|false|
```

Comment:
Is it OK, that you specified that parameter in this PR? 
If yes, please provide info about it in PR description

Reply:
Actually i didn't add this part this in this PR,  may be it came from Main when i pulled it and when i checked it was already there in main branch, so left it untoched.